openwisp_radius/api/views.py

```diff
@@ -1,8 +1,49 @@
+from django.contrib.auth.models import AnonymousUser
+from django.core.cache import cache
+from django_freeradius.api.views import AccountingView as BaseAccountingView
+from django_freeradius.api.views import AuthorizeView as BaseAuthorizeView
 from django_freeradius.api.views import BatchView as BaseBatchView
-from django_freeradius.api.views import accounting, authorize, postauth  # noqa
+from django_freeradius.api.views import PostAuthView as BasePostAuthView
+from django_freeradius.api.views import TokenAuthentication as BaseTokenAuthentication
+from rest_framework.exceptions import AuthenticationFailed
 
 from openwisp_users.models import Organization
 
+from ..models import OrganizationRadiusSettings
+
+
+class TokenAuthentication(BaseTokenAuthentication):
+    def authenticate(self, request):
+        id = None
```

Comment:
do you see github highlights `id`? That's because it's a built-in function.
Sometimes we are forced to use `id`, like when defining `id = UUIDField()`, but in this case we are not, so I think it's cleaner to use `uuid`, the syntax highlighting doesn't distract our mind when reading the code (remember code is read a lot more than it's written).